src/main/resources/application.properties

```diff
@@ -3,5 +3,7 @@ spring.datasource.username=postgres
 spring.datasource.password=postgres
 spring.datasource.driver-class-name=org.postgresql.Driver
 mybatis.mapper-locations=classpath:mapper/*.xml
+
+#mybatis.config.location=classpath:SqlMapConfig.xml
```

Comment:
How did we solve this? Is SqlMapConfig.xml being picked up by Spring-Boot now?

Reply:
no,  SqlMapConfig.xml is not being picked up by Spring-Boot now